.clang-format

```diff
@@ -1,47 +1,63 @@
-AccessModifierOffset: -3
-AlignEscapedNewlinesLeft: true
-AlignTrailingComments: true
-AllowAllParametersOfDeclarationOnNextLine: false
-AllowShortFunctionsOnASingleLine: true
-AllowShortIfStatementsOnASingleLine: false
-AllowShortLoopsOnASingleLine: false
-AlwaysBreakBeforeMultilineStrings: false
-AlwaysBreakTemplateDeclarations: true
-BinPackParameters: false
-BreakBeforeBinaryOperators: false
+---
+AccessModifierOffset: '-3'
+AlignAfterOpenBracket: Align
```

Comment:
I wonder if we should switch to `AlwaysBreak` - the hanging indent style ends up with everything far off to the right of the window, which can get a bit annoying.

Reply:
with `AlwaysBreak` it also looks ugly :)